### ISSUE TITLE
jinjava // either buggy or unimplemented

### ISSUE THREAD

**Issue by prior** [HubSpot/jinjava#169]
from [jinja math docs](http://jinja.pocoo.org/docs/2.10/templates/#math):
> //
Divide two numbers and return the truncated integer result. {{ 20 // 7 }} is 2.

However attempting this with jinjava throws an exception.  For example, I would expect this template:
```
20 // 7 = {{ 20 // 7 }}
```
to return `20 // 7 = 2`, but instead it throws this:
```
com.hubspot.jinjava.interpret.InterpretException: Error resolving expression [20 // 7]: NullPointerException: 
	at com.hubspot.jinjava.el.ExpressionResolver.resolveExpression(ExpressionResolver.java:89)
	at com.hubspot.jinjava.interpret.JinjavaInterpreter.resolveELExpression(JinjavaInterpreter.java:397)
```
```
Caused by: java.lang.NullPointerException
	at jinjava.de.odysseus.el.tree.impl.Parser.mul(Parser.java:547)
	at com.hubspot.jinjava.el.ext.ExtendedParser.add(ExtendedParser.java:149)
	at jinjava.de.odysseus.el.tree.impl.Parser.cmp(Parser.java:462)
```

**Comment by boulter** [comment]
Looks like this works for variables as params, just not literals. I'm looking at fixing.

**Comment by boulter** [comment]
Also cc @markus-s24

**Comment by bkrainer** [comment]
fixed in #234.
